include/nix/DataArray.hpp

```diff
@@ -313,6 +313,18 @@ class NIXAPI DataArray : public base::EntityWithSources<base::IDataArray>, publi
         }
         return backend()->createRangeDimension(backend()->dimensionCount() + 1, ticks);
     }
+    
+    /**
+     * @brief Append a new RangeDimension that uses the data stored in this DataArray as ticks.
+     * This works only(!) if the DataArray in 1D and the stored data is numeric. An Exception 
```

Comment:
Maybe specify which exception will be thrown (i.e. `InvalidDimension` exception) so one can catch the correct one.